heidegger_index/views.py

```diff
@@ -80,6 +78,37 @@ class LemmaDetailView(DetailView):
     template_name = "lemma_detail.html"
     context_object_name = "lemma"
 
+    def get_object(self, queryset=None):
+        """
+        CRUDE COPY OF get_object()
+        """
+        if queryset is None:
+            queryset = self.get_queryset()
+
+        slug = self.kwargs.get(self.slug_url_kwarg)
+        gnd = self.kwargs.get('gnd')
+        urn = self.kwargs.get('urn')
+        if slug is not None:
+            slug_field = self.get_slug_field()
+            queryset = queryset.filter(**{slug_field: slug})
+        if gnd is not None and slug is None:
```

Comment:
Hier een ander voorstel voor het laden van het juiste lemma a.d.h.v. de `gnd` of `urn` in de url. Ik weet nou niet of dit nu veel netter is.